### ISSUE TITLE
class websites deploying without student or tech data

### ISSUE THREAD

**Issue by joshdbarton** [nashville-software-school/class-website-template#1]
When deploying a new cohort site, the site does not populate with student or tech data. the template now appears to be missing its deploy file.

**Comment by drteresavasquez** [comment]
[This commit](https://github.com/nashville-software-school/class-website-template/commit/4ad4d1aee8c07ba47bbe2c521849ad497b46684f) by @stevebrownlee removes the deploy workflow and adds several assets and dependencies on the `main` branch. Not sure if this was intentional.

We need to get the dev and main branches in sync and test. [This was the solution](https://github.com/NSS-Day-Cohort-65/NSS-Day-Cohort-65.github.io/pull/1) to resolving the E65 deploy issue, but a PR shows there are many other dependencies that may need testing; however, it may have been a quick fix.

**Comment by hannahhall** [comment]
Just jumping in here, the deploy.yml file has to be present for it to deploy to github pages. Also the dev and main branches won't match because the gh-pages package is bundling all the files on the develop branch so the code can be deployed to github pages.

**Comment by drteresavasquez** [comment]
@hannahhall so to clarify when making simple updates to the UI, we should be doing that against which branch?

**Comment by drteresavasquez** [comment]
Never mind. I see on the removed readme to do it against develop and then run the deploy command.

**Comment by hannahhall** [comment]
Yeah and then the deploy command is really only needed if you're making changes to a class's actual repo. Deploying this repo doesn't matter because it's just the template that the class repo will copy and then build

**Comment by drteresavasquez** [comment]
Yeah. It was in reference to separate repos just in case so it all make sense. Thank you!!
